arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetwork.storyboard

```diff
@@ -16,22 +18,28 @@
                         <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="pzi-IO-kfE" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="44" width="414" height="769"/>
+                            </view>
```

Comment:
Adjusted the order of map view and statusLabel so that the label is on top of the map.